src/nxdoc/web-ui/web-ui-functional-testing.md

```diff
@@ -38,7 +38,7 @@ Feature: Login / Logout
 Please check the [Gherkin reference](https://docs.cucumber.io/gherkin/reference/) page for more details on how to write Cucumber tests.
 {{/callout}}
 
-Steps used in feature files are mapped into JavaScript instructions, which are read from **step definition** files. These are JavaScript files that map a given set of steps into JavaScript code. These definitions usually rely on page helpers to access interface elements and assert the conditions matching the test criteria. **Page helpers** are objects that encapsulate the logics required to access and manipulate high level interface elements, hiding most of the WebDriver-specific logics.
+Steps used in feature files are mapped into JavaScript instructions, which are read from **step definition** files. These are JavaScript files that map a given set of steps into JavaScript code. These definitions usually rely on page helpers to access interface elements and assert the conditions matching the test criteria. **Page helpers** are objects that encapsulate the logic required to access and manipulate high-level interface elements, hiding most of the WebDriver-specific logics.
```

Comment:
Maybe `**Page helpers** are objects that encapsulate the required logic to...`, wdyt?